misc/docker/base_images/debian_7_wheezy_32bit/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM obspy/base-images:debian_7_wheezy_32bit
+
+MAINTAINER Tobias Megies
+
+# Set the env variables to non-interactive
+ENV DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical DEBCONF_NOWARNINGS=yes
+
+# install packages to install obspy and build deb packages
+RUN apt-get update && apt-get upgrade -y || true
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    debhelper \
+    devscripts \
+    equivs \
+    fakeroot \
+    gcc \
+    gfortran \
+    git \
+    help2man \
+    lintian \
+    locales \
+    lsb-release \
+    python \
+    python-decorator \
+    python-dev \
+    python-imaging \
+    python-lxml \
+    python-matplotlib \
+    python-mock \
+    python-mpltoolkits.basemap \
+    python-mpltoolkits.basemap-data \
+    python-nose \
+    python-numpy \
+    python-pip \
+    python-requests \
+    python-scipy \
+    python-setuptools \
+    python-sqlalchemy \
+    python-suds \
+    python-support \
+    python-tornado \
```

Comment:
There was some hickup with matplotlib at some point (it was erroneously marked as a dependency at runtime in some deb package versions, see matplotlib/matplotlib#3039). I'd rather leave it in to avoid problems..